notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>If the imports are explained in the Markdown cell above, do we also need commenting here? A stylist question for Thomas. But then in that case this code block could be a lot shorter. And in the Markdown cell above, I think some of these can be grouped - e.g. explain keras in one bullet point, though there are multiple lines of code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Also, I need %matplotlib inline when in TIKE to get plots to display, is that standard?</p>

Reply:
<ul><li>In theory, you don't need the code comments if you have things in the markdown above. We could probably condense the imports into broader categories like "machine learning packages"</li></ul><p> </p><ul><li>%matplotlib inline is standard, some packages glitch if you don't do the magic command</li></ul>